doc/command-line-usage.md

```diff
@@ -0,0 +1,227 @@
+to the consumer. The current focus for the support tool is to provide features
+to encourage/support maintainers to add the package support info to their packages.
+In the future more advanced options can be added to the show command to provide 
+more deduplicated or focussed views of the information available.
+
+The general flow is:
+
+* install node.js (version 10.x or later) and ensure it is on the path
+* npm install the package support tool global (or optionally locally in which
+  case the `bin` directory need to be add the current path)
+* npm install the package to be assessed
+* cd into the directory which contains the package to be assessed
+* execute `support show`
+
+For example once node.js and the package support tool are installed:
+
+```shell
+npm install support
+cd node_modules/support
+support show
+```
+
+with a stripped down version of the the output to make it more easily readable being:
+
+```shell
+  @pkgjs/support(0.0.2) - { "versions": [ { "version": "*", "target": { "node": "supported" }, "response": { "type": "best-effort" }, "backing": { "hobby": "https://github.com/pkgjs/support" } } ] }
+    @npmcli/arborist(0.0.0) - unknown
+      @npmcli/installed-package-contents(1.0.5) - unknown
+        npm-bundled(1.1.1) - unknown
+        npm-normalize-package-bin(1.0.1) - unknown
+      @npmcli/map-workspaces(0.0.0-pre.1) - unknown
+        glob(7.1.6) - unknown
+          fs.realpath(1.0.0) - unknown
+          inflight(1.0.6) - unknown
+            wrappy(1.0.2) - unknown
+
+...
+
+      y18n(4.0.0) - unknown
+      yargs-parser(18.1.3) - unknown
+        camelcase(5.3.1) - unknown
+```
+
+The command shows the support information for the tree of packages starting from the package being assessed. 
+In this case it ONLY shows support info for the `support` package as the support information has not
+been added to any of the dependent packages yet.
+
+The support info shown is:
+
+```shell
+  @pkgjs/support(0.0.2) - { "versions": [ { "version": "*", "target": { "node": "supported" }, "response": { "type": "best-effort" }, "backing": { "hobby": "https://github.com/pkgjs/support" } } ] }
+```
+
+For the rest of the modules it simply shows `unknown` as the packages do not yet provide it.
+
+The package support tool prefers support information which is available locally within the package after installation.
+This is to allow offline use. The documented best practices for adding support info are written to maximize
+the likelyhood that at reasonably up to date copy of the support information is provided as part of the npm
+install.
+
+If the consumer wishes to ignore the local information and always use the canonical version of the support information
+they can add the `--canonical` option to the command. Instead of using the local support info, if available, it will
+display the URL from which the canonical information can be retrieved. This is the same behaviour when 
```

Comment:
```suggestion
display the URL from which the canonical information can be retrieved. This is the same behavior when 
```
unless we have a standard of using british spelling?

Reply:
No that's just the Canadian in me.  The standard is US english :)